cmds/tui.c

```diff
@@ -23,11 +23,16 @@
 #include "utils/dwarf.h"
 
 #define KEY_ESCAPE  27
-#define BLANK  32
+#define BLANK       32
+
+#define MAX_LINE_WIDTH 8192
 
 static bool tui_finished;
 static bool tui_debug;
 
+static char linebuf[MAX_LINE_WIDTH];
+static int field_width;
```

Comment:
I'd like to avoid these static variables for this case.

Reply:
It requires to modify the prototype of `print` field in `struct display_field` to avoid this.